src/lib/ngsi10/UpdateContextRequest.cpp

```diff
@@ -129,13 +128,13 @@ void UpdateContextRequest::fill
   const std::string&                 entityType
 )
 {
-  ContextElement* ceP = new ContextElement();
+  Entity* eP = new Entity();
 
-  ceP->entityId.fill(entityId, entityType, "false");
+  eP->fill(entityId, entityType, "false");
 
-  ceP->contextAttributeVector.fill((ContextAttributeVector*) &ucerP->contextAttributeVector);
+  eP->attributeVector.fill(ucerP->contextAttributeVector);
 
-  contextElementVector.push_back(ceP);
+  contextElementVector.push_back(eP);
```

Comment:
Right. However, would be a big change for this PR. I have added a FIXME #1298 mark about this, which ensures it will be done in a next PR, coming soon.

Fixed in 58d9889